README.md

```diff
@@ -80,13 +80,13 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 |
-|---------------------|----------------|----------------|----------------|----------------|
-| client-go 1.4       | +              | ✓              | -              | -              |
-| client-go 1.5       | +              | +              | -              | -              |
-| client-go 2.0       | +              | +              | ✓              | -              |
-| client-go 3.0 beta  | +              | +              | +              | ✓              |
-| client-go HEAD      | +              | +              | +              | +              |
+|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|---------------------|----------------|----------------|----------------|----------------|----------------|
+| client-go 1.4       | +              | ✓              | -              | -              | -              |
+| client-go 1.5       | +              | +              | -              | -              | -              |
+| client-go 2.0       | +              | +              | ✓              | -              | -              |
+| client-go 3.0       | + †            | +              | +              | ✓              | -              |
```

Comment:
@caesarxuchao I pushed some changes to your branch -- take a look and see if this still represents what you were trying to communicate.